pkg/ovs/openflow/ofctrl_bridge_test.go

```diff
@@ -160,3 +162,41 @@ func TestOFBridgePacketRcvd(t *testing.T) {
 		}
 	}
 }
+
+func TestOFMeterStats(t *testing.T) {
+	// Reset OVS metrics (Prometheus) and reinitialize them to test.
+	legacyregistry.Reset()
+	metrics.InitializeOVSMetrics()
+
+	b := NewOFBridge("test-br", GetMgmtAddress(ovsconfig.DefaultOVSRunDir, "test-br"))
+	sw := newFakeOFSwitch(b)
+	b.SetOFSwitch(sw)
+
+	mpMeterStatsReply := openflow15.NewMpReply(openflow15.MultipartType_MeterStats)
+	newMeterStats := func(meterID uint32, packets uint64) *openflow15.MeterStats {
+		meterStats := openflow15.NewMeterStats(meterID)
+		bandStats := openflow15.NewMeterBandStats()
+		bandStats.PacketBandCount = packets
+		meterStats.AddBandStats(*bandStats)
+		return meterStats
+	}
+	meterStats1 := newMeterStats(1, 100)
+	meterStats2 := newMeterStats(2, 0)
+	mpMeterStatsReply.Body = append(mpMeterStatsReply.Body, meterStats1)
+	mpMeterStatsReply.Body = append(mpMeterStatsReply.Body, meterStats2)
+	packetCounts := make(map[int]int64)
+	checkMeterStatsReply := func(meterId int, packetCount int64) {
+		packetCounts[meterId] = packetCount
+	}
+	b.GetMeterStats(checkMeterStatsReply)
+	// To make reply has the same Xid as request which is constructed with GetMeterStats
+	b.mpReplyChsMutex.RLock()
+	for k := range b.mpReplyChs {
+		mpMeterStatsReply.Xid = k
+	}
+	b.mpReplyChsMutex.RUnlock()
+	b.MultipartReply(sw, mpMeterStatsReply)
+	assert.Eventually(t, func() bool {
+		return packetCounts[1] == int64(100) && packetCounts[2] == int64(0)
+	}, 5*time.Second, time.Second)
```

Comment:
```suggestion
	}, 2*time.Second, 50*time.Millisecond)
```
1s interval would waste test time.

Reply:
Done.